insights/parsers/netstat.py

```diff
@@ -478,6 +478,7 @@ def parse_content(self, content):
         self.data = dict((s.name, s._merge_data_index()) for s in sections)
         self.lines = dict((s.name, s.lines) for s in sections)
         self.datalist = dict((s.name, s.datalist) for s in sections)
+        self.dataobjs = dict((s.name, s) for s in sections)
```

Comment:
Could you please rename this attribute as `self._dataobjs` to indicate it's a private attribute? since it's only used internally? Or, you may add it to the above `Attributes` section.

Reply:
Sure thing - done.